### DISCUSSION TITLE
Mono vs Stereo Mix

### DISCUSSION THREAD

**Discussion by grender007** [pschatzmann/arduino-audio-driver#15] [Q&A] [answered]
Hi!
First at all, my sketch with the Feather ESP32 S3 (as a sink) and the ES8388 (as ADC) is finally working!! There is only one thing I'm not sure how to do. I would like to use the Mono-Mix-to-Left-ADC mode of the ES8388 (I think the Stereo mix is the default option), but I think it is not defined in the library. I read the documentation, but I couldn't find a function to configure the board in this way. 
Is it included yet?

**Comment by pschatzmann** [comment, marked as answer]
This library implements only the functionality which is usually available across all audio chips. 
However you have the flexibility to use the register read and update API which is specific to your chip and implement any additional functionality yourself.
After callig begin(), you can update your desired register values yourself in your sketch using es8388_write_reg()
